modules/migration-mtc-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-4_{context}"]
+= {mtc-full} 1.8.4 release notes
+
+[id=technical-changes-1-8-4_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following technical changes:
+
+* {mtc-short} 1.8.4 extends its dependency resolution to include support for using {oadp-first} 1.4.
+
+[id="resolved-issues-1-8-4_{context}"]
+== Resolved issues
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following major resolved issues:
+
+.UI stuck at Namespaces while creating a migration plan
+
+When trying to create a migration plan from the {mtc-short} UI, the migration plan wizard becomes stuck at the *Namespaces* step. This issue has been resolved in {mtc-short} 1.8.4. link:https://issues.redhat.com/browse/MIG-1597[(MIG-1597)]
+
+.Migration is stuck as Direct Volume Migration fails with an `InvalidPVCs` error
+
+When performing a migration, the migration becomes stuck at the Direct Volume Migration (DVM) step, and before finally failing. link:https://issues.redhat.com/browse/MIG-1595[(MIG-1595)]
+
+.Migration fails with error of no matches for kind Virtual machine in version kubevirt/v1
+
+During the migration of an application, all the necessary steps, including the backup, DVM, and restore, are successfully completed. However, the migration is marked as unsuccessful with the error message `no matches for kind Virtual machine in version kubevirt/v1`. link:https://issues.redhat.com/browse/MIG-1594[(MIG-1594)]
+
+.Direct Volume Migration fails when migrating to a namespace different from the source namespace
+
+On performing a migration from source cluster to target cluster, with target namespace different from the source namespace, the DVM fails. link:https://issues.redhat.com/browse/MIG-1592[(MIG-1592)]
+
+.Direct Image Migration does not respect label selector on migplan
+
+When using Direct Image Migration (DIM), if a label selector is set on the migration plan, DIM does not respect it and attempts to migrate all imagestreams in the namespace. link:https://issues.redhat.com/browse/MIG-1533[(MIG-1533)]
+
+
+[id="known-issues-1-8-4_{context}"]
+== Known issues
+
+{mtc-short} 1.8.4 has the following known issues:
+
+.Ansible Operator is broken when {VirtProductName} is installed
+
+There is a bug in the `python3-openshift` package that installing {VirtProductName} exposes, with an exception `ValueError: too many values to unpack` returned during the task. Workaround: Remove the package and run the following command:
+[source,terminal]
+----
+$ pip install -U openshift.
+----
+
+[IMPORTANT] 
+==== 
+Do not run:
+
+[source,terminal]
+----
+$ pip3.8 --no-cache install -U openshift
+----
+====
+
+link:https://issues.redhat.com/browse/OCPBUGS-38116[(OCPBUGS-38116)]
```

Comment:
I'm not really sure what/how this should be documented here.

```suggestion
There is a bug in the `python3-openshift` package that installing {VirtProductName} exposes, with an exception `ValueError: too many values to unpack` returned during the task. 

link:https://issues.redhat.com/browse/OCPBUGS-38116[(OCPBUGS-38116)]
```

While the root of the problem will need to be addressed via an ansible operator image update, we (MTC Eng.) have implemented a workaround in our 1.8.4 release and intend to carry it until the ansible operator image is fixed.  Upon updating to 1.8.4 (when shipped), MTC will no longer be affected by the issue described in OCPBUGS-38116.  It is not expected for installations of MTC <1.8.4 encountering this issue to perform any action beyond updating to >=1.8.4.